src/main.js

```diff
@@ -28,6 +30,8 @@ Apify.main(async () => {
     if (verboseLog) {
         log.setLevel(log.LEVELS.DEBUG);
     }
+
+    const minMaxDate = time.minMaxDates({ min: videosFromDate || new Date(Date.now()/1000) , max: videosToDate || new Date()});
```

Comment:
can leave `undefined`, it won't use it
`minMaxDates({ min: undefined, max: undefined }).compare()` effectively is like a passthrough